app/src/main/java/com/telnyx/webrtc/sdk/ui/MainViewModel.kt

```diff
@@ -88,8 +88,13 @@ class MainViewModel @Inject constructor(
         telnyxClient?.call?.acceptCall(callId, destinationNumber)
     }
 
-    fun endCall(callId: UUID) {
-        telnyxClient?.call?.endCall(callId)
+    fun endCall(callId: UUID? = null) {
+        callId?.let {
+            telnyxClient?.call?.endCall(callId)
+        } ?: run {
```

Comment:
Beautiful use of scoped functions here 👍

Reply:
Thanks! But I'm still learning Kotlin and ways to do less/more efficient code